specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/file.json

```diff
@@ -410,6 +448,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/fileServices/default/shares/{shareName}/restore": {
+      "post": {
+        "tags": [
+          "FileShares"
+        ],
+        "operationId": "FileShares_Restore",
+        "x-ms-examples": {
+          "PutShares": {
```

Comment:
RestoreShares maybe better here.

Reply:
you are right! good catch here. I might copy it from PutShares and forgot to change it.